src/core/server/legacy/legacy_service.ts

```diff
@@ -69,14 +72,21 @@ export interface LegacyServiceStartDeps {
   plugins: Record<string, unknown>;
 }
 
+export interface LegacyServiceSetup {
+  pluginSpecs: unknown[];
```

Comment:
does someone consume it?

Reply:
I've marked this as internal, but also fleshed out the interface.